src/main/java/fi/tkgwf/ruuvi/common/parser/impl/DataFormat5Parser.java

```diff
@@ -39,7 +39,7 @@ public RuuviMeasurement parse(byte[] data) {
         if (!ByteUtils.isMaxSignedShort(data[9], data[10])) {
             m.setAccelerationY((data[9] << 8 | data[10] & 0xFF) / 1000d);
         }
-        if (!ByteUtils.isMaxSignedShort(data[11], data[2])) {
+        if (!ByteUtils.isMaxSignedShort(data[11], data[12])) {
```

Comment:
I think this is obvious typo as well. Tests would fail if this is not changed.

Reply:
oh, yes it is :) and this is why we (should) have tests..